### DISCUSSION TITLE
For those who have securly

### DISCUSSION THREAD

**Discussion by SIMONHSCH** [3kh0/ext-remover#1125] [Polls JOIN TITANIUM NETWORK TO SHARE PLEASE]
I' asking that because recently, my school switched from `knneimmpeamikmjijmnhjmmobddleohj` to `iheobagjkfklnlikgihanlhcddjoihkg` (I don't know why did they choose the old Securly id)

**Comment by S-PScripts** [comment]
iheobagjkfklnlikgihanlhcddjoihkg:

Securly for Chromebooks:
https://chromewebstore.google.com/detail/securly-for-chromebooks/iheobagjkfklnlikgihanlhcddjoihkg?hl=en


List of extensions: https://extensions.securly.com/extensions.xml
